Data/ByteString.hs

```diff
@@ -1020,7 +1020,7 @@ dropWhileEnd :: (Word8 -> Bool) -> ByteString -> ByteString
 dropWhileEnd f ps = unsafeTake (findFromEndUntil (not . f) ps) ps
 {-# INLINE dropWhileEnd #-}
 
--- instead of findIndexOrEnd, we could use memchr here.
+-- instead of findIndexOrLength, we could use memchr here.
```

Comment:
I don't understand this comment. I thought it might suggest that `break (== x)` should use `memchr`, but that was already done when the comment was added in https://github.com/haskell/bytestring/commit/ea619ea39469ba50599f58162a1fc55625c892bf.

Reply:
Could it have possibly originally refered to `dropWhile`, before `dropWhileEnd` was added?.. Let's remove this line, it does not make sense to me as well.

Reply:
> Could it have possibly originally refered to `dropWhile`, before `dropWhileEnd` was added?

Yeah, that makes sense.

> Let's remove this line, it does not make sense to me as well.

Done!